Berlin/BerHsOr9834.xml

```diff
@@ -60,11 +60,11 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
                   <decoDesc>
                      <decoNote type="miniature" xml:id="d1">
                        <locus target="#1v"></locus>
-                        <desc>Miniature of <persName ref="PRS1962Anthony">ʾƎnṭonǝs</persName>.</desc>
+                        <desc>Miniature of <ref type="authFile" corresp="AT1035HolyMan"> unknown Saint.</desc>
                      </decoNote>
                      <decoNote type="miniature" xml:id="d2">
                        <locus target="#2r"></locus>
-                        <desc>Miniature of unknown Saint.</desc>
+                        <desc>Miniature of <ref type="authFile" corresp="AT1035HolyMan"> saint <persName ref="PRS1962Anthony">ʾƎnṭonǝs</persName>.</desc>
```

Comment:
the `<ref>` tag must be closed

if your automplete is installed on atom actually if you are not too fast and just type `<` to open a tag and wait for suggestions, then select a suggested tag and click enter, it will type `<ref></ref>`, so open and close for you automatically, and you do not have to worry